src/Plugin/TfaValidation/TfaTotp.php

```diff
@@ -41,6 +41,8 @@ class TfaTotp extends TfaBasePlugin implements TfaValidationInterface {
    */
   protected $alreadyAccepted;
 
+  protected $context;
```

Comment:
Good find, I think I am going to change everything labeled $context to $configuration to correctly match the plugin terminology. Not ready to do it in this PR though.